changelogs/2023-05-10-feature-core-api.md

```diff
@@ -0,0 +1,33 @@
+### Added
+- [ChronAm JSON API] at /api/chronam/
+  - Original endpoints were [changed to JSON-LD IIIF
+    responses](https://github.com/open-oni/open-oni/pull/127). Original
+    responses are now available again. Read more about Open ONI API resources in
+    the [About API markup](/core/templates/about_api.html).
+  - Now powered by [Django Rest Framework] which provides throttling, auth, etc
+  - Include new error checking and testing
+  - Update `/about/api/` text & `<link rel="alternate">` links on relevant pages
+  - Throttling of the API by IP can be enabled with the environment variable
+    `ONI_CHRONAM_API_THROTTLE`. Read more in [configuration
+    documentation](/docs/customization/configuration.md#onisitesettings_basepy)
+
+[ChronAm JSON API]: https://chroniclingamerica.loc.gov/about/api/#json-views
+[Django Rest Framework]: https://www.django-rest-framework.org/
+
+### Migration
```

Comment:
That makes sense. In the smaller changelog file for just this PR the mention of the setting is hard to miss, but once the others are combined it might be overlooked. Think I should just move this part from the Added section to Migration?

> Throttling of the API by IP can be enabled with the environment variable
    `ONI_CHRONAM_API_THROTTLE`. Read more in [configuration
    documentation](https://github.com/open-oni/open-oni/blob/feature/core-api/docs/customization/configuration.md#onisitesettings_basepy)

I defaulted to not throttling so it's not changing from how it originally behaved in the past out of the box. And it'd be presumptuous to guess what a "reasonable" default for throttling would be for a variety of ONI sites.

Reply:
I also want to add a note to Migration about updating `docker-compose.yml` and `.env` if one is working with ONI in that context.

Reply:
Ooof, I don't <3 github comments sometimes. I commented on the "### Migration" line but the github review shows context *above* instead of around or just below that.

Pretend this is what I wrote: "the new setting should be mentioned specifically in the migration section so people migrating will know they should set (or at least be aware of) any new settings."

Reply:
I grokked that after wondering why would Jeremy point at those links with the comment. It is dumb that it only shows code context in one direction from where you apply the comment.

Yep, I'll move it to the Migration section :+1: Was just curious if you thought I should leave it in both or if I should add anything to it in Migration. I'll copy the default / examples from the docs page to go with it in Migration too.

Reply:
Gotcha - I'm good with either approach just so long as it's in the migration at a minimum.

Reply:
Let me know what you think of the updated changelog and API root path self-description which it mirrors.

Reply:
I think it's good, thanks!